src/gccrs/args.rs

```diff
@@ -63,7 +63,7 @@ fn format_output_filename(
     match crate_type {
         CrateType::Bin => output_file.push(&format!("{}{}", crate_name, extra_filename)),
         CrateType::DyLib => output_file.push(&format!("lib{}.so{}", crate_name, extra_filename)),
-        CrateType::StaticLib => output_file.push(&format!("lib{}.a{}", crate_name, extra_filename)),
+        CrateType::StaticLib => output_file.push(&format!("lib{}{}.a", crate_name, extra_filename)),
```

Comment:
Is this a dup of #18 ?

Reply:
Yup, exactly. This also needs to be acquired through a function or variable. The diff here is that the wrong filename was being created: Instead of `libexemple-173747474bc.a`, `libexemple.a-173747474bc` was created